### PR TITLE
start work on todo

### PR DESCRIPTION
**Pull request by anshul9867**
Make the UI of Todo

### CODE REVIEW

File: src/TodoApp/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { ReactDOM } from "react";
+import { Badge} from 'react-bootstrap';
+
+function Todo() {
+
+    const TODO_LIST = [
```
Comment (singharpan09) [inline src/TodoApp/index.js]: make a constants folder in src and make todolist.json there
 and use this from there

File: src/TodoApp/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <ul>
+                {
+                    TODO_LIST.map(val => {
+                        return(
+                            <div className="main">
+                                <div className="list">
+                            <input type='checkbox' checked={val.isdone}/>
+                            <li>{val.name}</li>
+                            <div>
+                            {val.isdone===true?<Badge bg="secondary" className="badge">Completed</Badge>:null}
```
Comment (singharpan09) [inline src/TodoApp/index.js]: is ===true needed ?

File: src/TodoApp/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    ]
+
+    return (
+        <div>
+            <ul>
+                {
+                    TODO_LIST.map(val => {
+                        return(
+                            <div className="main">
+                                <div className="list">
```
Comment (singharpan09) [inline src/TodoApp/index.js]: make separate component for todolist item and that just need to pass todo value

File: src/TodoApp/index.js (hunk 2 above)
Comment (anshul9867) [reply src/TodoApp/index.js]: done

File: src/TodoApp/index.js (hunk 1 above)
Comment (anshul9867) [reply src/TodoApp/index.js]: Done

**Pull request merged**